internal/pkg/daemon/checks.go

```diff
@@ -36,9 +36,13 @@ func (d *Daemon) preUpgradeChecks(
 
 	// notify once about the upcoming upgrade
 	if currStep == urproto.UpgradeStep_MONITORING {
+		chainName := "unsupported-name"
```

Comment:
what is the purpose of that one?

Reply:
We can't access nodeInfo if there's no gRPC, this dependency is not necessary, as we already ask for a unique network identifier in the config (`UpgradeRegistry.Network`)